Source/WebCore/platform/graphics/displaylists/DisplayListResourceHeap.h

```diff
@@ -92,23 +93,41 @@ class ResourceHeap {
         add<FontCustomPlatformData>(renderingResourceIdentifier, WTFMove(customPlatformData), m_customPlatformDataCount);
     }
 
-    ImageBuffer* getImageBuffer(RenderingResourceIdentifier renderingResourceIdentifier) const
+    ImageBuffer* getImageBuffer(RenderingResourceIdentifier renderingResourceIdentifier, OptionSet<RecordReplayOption> options) const
```

Comment:
`options` can be defaulted to `{ }` so non skia calls will not change.

Reply:
Ok, it was helpful during development, to assure I'm not misisng any callsite. Will adapt it now.

Reply:
Done for getSourceImage/getImageBuffer/getNativeImage.